torchci/clickhouse_queries/log_captures_count/query.sql

```diff
@@ -1,21 +1,31 @@
--- !!! Query is not converted to CH syntax yet.  Delete this line when it gets converted
+with jobs as (
+    select
+        j.torchci_classification.line as example,
```

Comment:
minor nit: Seems like it would make line 17 clearer below, (`any(line) as example`)

```suggestion
        j.torchci_classification.line as line,
```